components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -703,8 +703,11 @@
             if (owner &amp;&amp; ownershipTransfer) {
               var newOwner = '';
               var newOwnerGroup = ownershipTransfer.down('select');
+              var newOwnerURL='/bin/';
```

Comment:
No, that's not guaranteed to be there. See phenomecentral.org for example. And even if it is there, it could be renamed to something more user-friendly.